platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationService.java

```diff
@@ -36,4 +37,11 @@
      * @return MigrationWarning returned if there were non-fatal issues when exporting
      */
     Collection<MigrationWarning> export(@NotNull Path exportDirectory) throws MigrationException;
+
+    /**
+     * Get a collection of the migratables available for export
```

Comment:
How about `Gets detailed information about all the {@link DataMigratable} services currently registered`?